src/components/sidebar/routePathView/routePathListTab/RoutePathListNode.tsx

```diff
@@ -265,14 +317,12 @@ class RoutePathListNode extends ViewFormBase<
 
     private renderNodeView = (node: INode) => {
         return (
-            <div className={s.nodeContent}>
-                <div className={s.flexRow}>
-                    <TextContainer
-                        label='MITTAUSPÄIVÄMÄÄRÄ'
-                        value={node.measurementDate}
-                        darkerInputLabel={true}
-                    />
-                </div>
+            <div className={s.flexRow}>
+                <TextContainer
+                    label='MITTAUSPÄIVÄMÄÄRÄ'
+                    value={node.measurementDate}
```

Comment:
measurmentDate is in this version string, and that is why this is broken. My PR has it typed to Date, so it should fix the formatting issue